nautobot_ui/README.md

```diff
@@ -0,0 +1,29 @@
+# Nautobot v2 UI Prototype
+
+## Running it
+
+Install and compile dependencies:
+
+```no-highlight
+nautobot-server build
+```
+
+Configure Nautobot to pass through authentication over CORS by putting this into your `nautobot_config.py`:
+
+```python
+CORS_ALLOW_ALL_ORIGINS = True
+CORS_ALLOW_CREDENTIALS = True
+SESSION_COOKIE_SAMESITE = None
+```
```

Comment:
Methinks this is no longer necessary since the web UI has basic (even if somewhat flaky) session authentication.